Google.Api.Generator.Tests/ProtoTests/RoutingHeadersExplicit/RoutingHeadersExplicit.proto

```diff
@@ -0,0 +1,121 @@
+﻿syntax = "proto3";
+
+package testing.routingheadersexplicit;
+
+option csharp_namespace = "Testing.RoutingHeadersExplicit";
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/routing.proto";
+
+service RoutingHeadersExplicit {
+	option (google.api.default_host) = "routingheaders.example.com";
```

Comment:
I believe all our other proto files use spaces instead of tabs (and two-space indent).